<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="m_mainWindowGridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>452</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="7" column="1">
         <widget class="QDoubleSpinBox" name="m_gridsize">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QSpinBox" name="m_right">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Force</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QSpinBox" name="m_bottom">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="m_resolutionY">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QPushButton" name="m_stop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QPushButton" name="m_initialise">
          <property name="text">
           <string>Initialise</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QSpinBox" name="m_resolutionX">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QSpinBox" name="m_top">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Grid Size</string>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QPushButton" name="m_start">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QSpinBox" name="m_left">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QDoubleSpinBox" name="m_timestep">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="20" column="1">
         <widget class="QPushButton" name="m_step">
          <property name="text">
           <string>Step</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="4">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Initial Fluid Position</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="m_pre">
          <property name="text">
           <string>Pressure</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="m_vel">
          <property name="text">
           <string>Velocity</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QDoubleSpinBox" name="m_forceY">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
          <property name="value">
           <double>-9.810000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Time Step</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QDoubleSpinBox" name="m_forceX">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="m_flu">
          <property name="text">
           <string>Fluid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="m_type">
          <item>
           <property name="text">
            <string>FLIP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PIC</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_type</tabstop>
  <tabstop>m_flu</tabstop>
  <tabstop>m_vel</tabstop>
  <tabstop>m_pre</tabstop>
  <tabstop>m_resolutionX</tabstop>
  <tabstop>m_resolutionY</tabstop>
  <tabstop>m_gridsize</tabstop>
  <tabstop>m_timestep</tabstop>
  <tabstop>m_forceX</tabstop>
  <tabstop>m_forceY</tabstop>
  <tabstop>m_right</tabstop>
  <tabstop>m_left</tabstop>
  <tabstop>m_top</tabstop>
  <tabstop>m_bottom</tabstop>
  <tabstop>m_initialise</tabstop>
  <tabstop>m_step</tabstop>
  <tabstop>m_start</tabstop>
  <tabstop>m_stop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
